### DISCUSSION TITLE
[Bug]: Intermittent boot loop on Loratec v3 (but probably just bad hardware)

### DISCUSSION THREAD

**Discussion by vext01** [meshtastic/firmware#4481] [Troubleshooting]
### Category

Other

### Hardware

Heltec V3

### Firmware Version

firmware-heltec-v3-2.4.1.394e0e1.bin

### Description

Hi,

I suspect this is bad hardware, but since I'm new to MT, I'm not sure and I wasn't sure where else to ask. Apologies if this is the wrong place to post.

I bought a Heltec v3 and flashed `firmware-heltec-v3-2.4.1.394e0e1.bin` (I've tried via both the web flasher and using esptool).

Once flashed, I set up the device and it seemed to work just fine. One time I had the device running for 4 hours just fine. But eventually it starts boot-looping intermittently. Sometimes it manages to work itself out of a boot loop, but eventually it always comes back.

Looking at the serial output I see:
```
...
DEBUG | ??:??:?? 1 [Power] Battery: usbPower=0, isCharging=0, batMv=3100, batPct=0
DEBUG | ??:??:?? 1 [Screen] Screen: Started...
DEBUG | ??:??:?? 1 [Screen] haveGlyphs=1
INFO  | ??:??:?? 1 [RangeTestModule] Range Test Module - Disabled
DEBUG | ??:??:?? 2 [Screen] haveGlyphs=1
DEBUG | ??:??:?? 3 [Screen] haveGlyphs=1
DEBUG | ??:??:?? 4 [Screen] haveGlyphs=1
INFO  | ??:??:?? 4 [PowerFSM] Loss of power in Powered
INFO  | ??:??:?? 4 [PowerFSM] Loss of power in Powered
INFO  | ??:??:?? 4 [PowerFSM] Initialise the NimBLE bluetooth module
DEBUG | ?ESP-ROM:esp32s3-20210327
Build:Mar 27 2021
rst:0xf (BROWNOUT_RST),boot:0x8 (SPI_FAST_FLASH_BOOT)
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fce3808,len:0x4bc
load:0x403c9700,len:0xbd8
load:0x403cc700,len:0x2a0c
entry 0x403c98d0
E (335) esp_core_dump_flash: No core dump partition found!
E (335) esp_core_dump_flash: No core dump partition found!
�@INFO  | ??:??:?? 0

//\ E S H T /\ S T / C

INFO  | ??:??:?? 0 Booted, wake cause 0 (boot count 1), reset_reason=brownout
DEBUG | ??:??:?? 0 Filesystem files (507904/1048576 Bytes):
DEBUG | ??:??:?? 0  /prefs/channels.proto (57 Bytes)
DEBUG | ??:??:?? 0  /prefs/config.proto (99 Bytes)
...
```

This suggests brownout. But I've tried powering the device using all kinds of different USB A->C cables using all kinds of different sources (computers, GAN chargers, power banks, wall warts,  ...).

Sounds like anything you've seen before? Is there anything I can try? Does this sound like bad hardware to you? Shall I return it?

Cheers

### Relevant log output

```Shell
(see above)
```

**Comment by vext01** [comment]
Region is `EU_868`, in case that matters. I'm in the UK.

**Comment by thebentern** [comment]
The brownout does make it appear to be hardware failure perhaps. Sometimes that can indicate an LDO failure because it is not capable of producing a consistent 3.3V power supply to the MCU. Do you have another Heltec V3 to try?

**Reply by vext01** [reply]
I have another on the way from AliExpress. Looks like I may have to wait and see if you have no other ideas.

**Comment by vext01** [comment]
The second V3 arrived and I've had no issues with it, so I think I just got a bad one.

**Comment by thebentern** [comment]
Thanks for the follow up. Glad the new one is working
